eslint.config.mjs

```diff
@@ -0,0 +1,36 @@
+import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
+
+export default [
```

Comment:
so i _think_ the 0.0.1-alpha.6 way could simplify to:
```javascript
export default hmppsConfig({
  extraIgnorePaths: ['test_results'],
  extraPathsAllowingDevDependencies: ['playwright.config.ts'],
  extraGlobals: { $: false },
})
```

but i've not tried it

Reply:
yes, I came up with precisely that 😄